server/src/main.rs

```diff
@@ -21,7 +21,12 @@ mod tables;
 mod types;
 mod storage;
 
-use std::sync::Arc;
```

Comment:
There should only be one newline between modules and imports.

Reply:
Fixed